usecases/usecase/usecase.go

```diff
@@ -198,19 +219,38 @@ func (u *UseCaseBase) updateRemoteEntityScenarios(
 	}
 }
 
+// remove all remote entities of a device from the use case
+func (u *UseCaseBase) removeDeviceFromAvailableEntityScenarios(device spineapi.DeviceRemoteInterface) {
+	indicies := u.entityScenarioIndicesOfDevice(device)
+
+	for i := range indicies {
+		u.removeEntityIndexFromAvailableEntityScenarios(i)
+	}
```

Comment:
Shouldn't this be ```for _, i := range indicies {```?
If I'm reading this correctly, entityScenarioIndicesofDevice can return non-consecutive indices (or indices not starting at 0) but this loop will always remove the first `n = len(indicies)` scenarios

Reply:
Yes, you are right!